package.json

```diff
@@ -39,7 +40,7 @@
     "carbon-components-react": "^7.7.0",
     "carbon-icons": "^7.0.7",
     "cross-env": "^5.1.4",
-    "cypress": "3.2.0",
+    "cypress": "3.8.1",
```

Comment:
Note: when I updated this setting, then re-ran `yarn`, it automatically downloaded a new Cypress binary (runs the Cypress UI), which took like 40 minutes. Subsequent `yarn` runs didn't download this extra piece, so it should be a one-time thing for devs. Just something to be aware of when checking out this branch.